new_airbnb.py

```diff
@@ -4,10 +4,16 @@
 from flask import Flask
 from flask_cors import CORS
 
+from utils.error_handler import generate
+
 app = Flask(__name__)
 app.secret_key = "".join(random.choice(string.ascii_letters + string.digits) for _ in range(15))
 CORS(app)
 
+error_codes = [400, 401, 403, 404, 405, 409, 413, 500]
```

Comment:
I think these codes would be better to be placed in error_msg.py so it's easier to check if every status code has a corresponding message.

Also registering these error handlers is not the job of this python file; its only purpose is to store `app` for other modules to import and we should keep this file from our business logic as far as possible.

I suggest we call `generate` in error_handler.py (as an initialization), and import error_handler in \_\_init\_\_ of views; error codes are more related to views than to the app itself logically.